recipes/transabyss/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -x -e
+
+TOPDIR=`pwd`
+
+mkdir -p $PREFIX/bin
+
+chmod 777 bin/*
+cp bin/* $PREFIX/bin
+cp -rf configs $PREFIX/
+
+mkdir -p $TOPDIR/scripts
+mv transabyss $TOPDIR/scripts/
+mv transabyss-merge $TOPDIR/scripts/
+
+cd $TOPDIR/scripts/
+2to3 -w *
+cd $TOPDIR/utilities
+2to3 -w *
+
+cd $TOPDIR
+cd utilities
+sed -i.bak 's/from utilities import/from transabyss import/' *.py
```

Comment:
If I want to keep it with python3, then yes.

I suppose I could build it with python2. I add a 

```
skip: True #[py35]
```

What do you think? 2to3 or just keep it as python 2?